contracts/src/v0.8/functions/dev/interfaces/IFunctionsOracle.sol

```diff
@@ -20,6 +20,21 @@ interface IFunctionsOracle {
   function setRegistry(address registryAddress) external;
 
   /**
+   * @notice Returns the DON's threshold encryption public key used to encrypt secrets
+   * @dev All nodes on the DON have separate key shares of the threshold decryption key
+   * 2f + 1 nodes must participate in a threshold decryption OCR round to decrypt secrets
```

Comment:
I'd remove the note about 2f+1 as this is configurable and can potentially change in the future.